test/test/alloc/AllocTests.java

```diff
@@ -50,6 +50,9 @@ public void startup(TestProcess p) throws Exception {
 
     @Test(mainClass = MapReaderOpt.class, agentArgs = "start,event=G1CollectedHeap::humongous_obj_allocate", jvmArgs = "-XX:+UseG1GC -XX:G1HeapRegionSize=1M -Xmx4g -Xms4g", os = Os.LINUX)
     public void humongous(TestProcess p) throws Exception {
+        if (System.getProperty("os.arch").equals("ppc64le")) {
+            return; // method hooks are not supported on PowerPC
+        }
```

Comment:
`arch` attribute of the `@Test` annotation is exactly for this purpose.
It may be more conveninet to add a different attribute to _exlucde_ particular architecture instead of specifying a list of included architectures.

Reply:
This could be pretty nice.